modules/tinymce/src/plugins/legacyoutput/test/ts/browser/LegacyOutputPluginTest.ts

```diff
@@ -1,152 +1,176 @@
-import { Log, Pipeline } from '@ephox/agar';
-import { UnitTest } from '@ephox/bedrock-client';
+import { describe, it } from '@ephox/bedrock-client';
 import { Cell } from '@ephox/katamari';
-import { LegacyUnit, TinyLoader } from '@ephox/mcagar';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/mcagar';
+import { assert } from 'chai';
+import Editor from 'tinymce/core/api/Editor';
 
-import Plugin from 'tinymce/plugins/legacyoutput/Plugin';
 import Theme from 'tinymce/themes/silver/Theme';
```

Comment:
This should be beside the `Plugin`/`Theme` import as it's an internal import. eg:
```
import { assert } from 'chai';

import Editor from 'tinymce/core/api/Editor';
import Plugin from 'tinymce/plugins/legacyoutput/Plugin';
import Theme from 'tinymce/themes/silver/Theme';
```

Reply:
updated